v2/stream/new/options.go

```diff
@@ -0,0 +1,144 @@
+package new
+
+import (
+	"os"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+)
+
+type options struct {
+	logger         Logger
+	host           string
+	key            string
+	secret         string
+	reconnectLimit int
+	reconnectDelay time.Duration
+	processorCount int
+	trades         []string
+	tradeHandler   func(Trade)
+	quotes         []string
+	quoteHandler   func(Quote)
+	bars           []string
+	barHandler     func(Bar)
+}
+
+func (o *options) apply(opts ...Option) {
+	for _, opt := range opts {
+		opt.apply(o)
+	}
+}
+
+type Option interface {
```

Comment:
Disagreed, `Option` is better in my opinion.

Reply:
Since this is a tie I'm not going to change it.

Reply:
This is against Effective Go: https://golang.org/doc/effective_go#interface-names

Reply:
The reason this name/solution was picked because using the Go gRPC SDK is so "natural". As you can see [they are using](https://github.com/grpc/grpc-go/blob/master/dialoptions.go#L76) the exact same naming convention and I think it works very well. Especially because the only method is unexported.